server/data_access_layer/daos/transac_stat.dao.js

```diff
@@ -96,6 +96,13 @@ exports.prepareDuesQuery = (yearMonthList, employeeId, clientType, countryLabel,
         }
     }
 
+    if(accountType !== undefined) {
+        whereString = whereString.concat(" AND ACS.CONNECTION_ID=7 ");
+    }
+    else {
+        whereString = whereString.concat(" AND ACS.CONNECTION_ID=3 ");
+    }
```

Comment:
I dont think you need to put the else here because we are already checking connection id 3 by default

Reply:
Yes since it is deleted in the base query